mbed_project/mbed_project.py

```diff
@@ -17,7 +17,14 @@ def clone_project(url: str, recursive: bool = False) -> None:
         url: URL of the repository to clone.
         recursive: Recursively clone all project dependencies.
     """
-    pass
+    git_data = parse_url(url)
+    url = git_data["url"]
+    dst_path = pathlib.Path(git_data["dst_path"])
+    click.echo(f"Cloning Mbed program from '{url}'")
```

Comment:
is tere a reason why the logger i.e. `logger.info()` is not used here and click is used instead?
The reason I am asking is that these functions look like APIs which could be used without being in the CLI/click context unless I am wrong.

Reply:
We need to print these messages to the user regardless of log level.

Reply:
Although I probably should just use `print` rather than the `click.echo` thing.

Reply:
I'll see if I can pull these messages up into the CLI code, that's probably the best option here.

Reply:
👍

Reply:
ec59d29